x-pack/plugins/ml/public/application/datavisualizer/file_based/components/fields_stats_grid/get_field_names.ts

```diff
@@ -29,3 +31,19 @@ export function getFieldNames(results) {
   }
   return tempFields;
 }
+
+export function getSupportedFieldType(type: string): MlJobFieldType {
+  switch (type) {
+    case 'long':
```

Comment:
we can use `ES_FIELD_TYPES` from the data plugin here.

Reply:
Updated here https://github.com/elastic/kibana/pull/87598/commits/47dcf86095f2e0f9e3d78b6cb4c5ebbf5c17cd94